.docker/docker-compose.yml

```diff
@@ -0,0 +1,15 @@
+version: '3.4'
+
+services:
+  app:
+    image: knp-trello-pu:app
+    build:
+      context: ../
+      dockerfile: .docker/app/Dockerfile
+    volumes:
+      - ./../:/app
```

Comment:
Sure about `./../` ?

Reply:
That volume has been removed, we dont need to mount it inside the docker container.